central/debug/service/service.go

```diff
@@ -55,6 +54,7 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
```

Comment:
If we're moving from gogo `Empty` type to the google well known type parent, I think it would be good to have #10151 merged before. This should reduce the number of non-generated files to review in this PR.